server/grpc_service.go

```diff
@@ -176,6 +183,49 @@ func (s *GrpcServer) Tso(stream pdpb.PD_TsoServer) error {
 		}
 
 		streamCtx := stream.Context()
+		if !s.IsTSOEnabled() {
```

Comment:
How about just using api mode?

Reply:
I am not sure whether we support downgrade in api service mode, when tso service is not available.

Reply:
Please change it